object_detection/README.md

```diff
@@ -35,7 +35,7 @@ Before You Start:
 * <a href='g3doc/installation.md'>Installation</a><br>
 
 Quick Start:
-* <a href='object_detection_tutorial.ipynb'>
```

Comment:
My thought was that the [`running_notebook.md`](https://github.com/tensorflow/models/blob/master/object_detection/g3doc/running_notebook.md) is not referred to anywhere. It seems to me that it is the brief intro to running the notebook (maybe for those who doesn't know how to run jupyter notebook locally), and it links to [`object_detection_tutorial.ipynb`](https://github.com/tensorflow/models/blob/master/object_detection/object_detection_tutorial.ipynb) anyway.

That said, `object_detection_tutorial.ipynb` would be viewable on GitHub, but in two clicks, which might not be preferred. What do you think?